frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/pipeline-metrics-utils.ts

```diff
@@ -119,13 +120,43 @@ export const getXaxisValues = (timespan: number): number[] => {
   return xValues.slice(0, numDays);
 };
 
-export const getYaxisValues = (seconds: number): string => {
-  if (seconds < 60) {
-    return `${seconds}s`;
+export const getDuration = (seconds: number, list?: boolean): string => {
```

Comment:
I would suggest a name that is independent of the use case "list" or "detail page" and use a more generic term like "short" or "long" here. Wdyt?

```suggestion
export const getDuration = (seconds: number, long: boolean): string => {
```